spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -97,8 +97,8 @@ def in_house?
       @in_house = (team_information['type'] == 'In-House')
     end
 
-    def platform_slug(mac)
-      if mac
+    def platform_slug(platform)
```

Comment:
this change seems to be the only one that is breaking API fully (i.e. parameter is fully replaced, not used as an optional one that could be kept for backward compatibility)